boringtun/src/noise/handshake.rs

```diff
@@ -231,8 +232,8 @@ impl std::fmt::Debug for HandshakeInitSentState {
     }
 }
 
-#[derive(Debug)]
-enum HandshakeState {
+#[derive(Debug, Clone)]
```

Comment:
Making this `pub` with a stock `Debug` impl exposes the `chaining_key` of both Init states, which we need to avoid.